lambdas/smartblocks-daily_put.ts

```diff
@@ -1,44 +0,0 @@
-import { APIGatewayProxyHandler } from "aws-lambda";
```

Comment:
We used to store whether or not Daily smartblocks ran in RoamJS pre-Roam Depot. No longer necessary